packages/cbioportal-utils/src/hotspot/CancerHotspotsUtils.ts

```diff
@@ -106,6 +106,15 @@ export function filter3dHotspotsByMutations(
     );
 }
 
+export function filterSpliceHotspotsByMutations(
+    mutations: Mutation[],
+    index: IHotspotIndex
+): Hotspot[] {
+    return filterHotspotsByMutations(mutations, index, (hotspot: Hotspot) =>
+        hotspot.type.toLowerCase().includes('splice')
+    );
+}
+
 export function isRecurrentHotspot(
```

Comment:
I know naming of this already existed, but might be worth revisiting. The 3d hotspots were also determined using recurrence, so maybe LinearHotspot or LinearClusterHotspot is better? Using terminology from Lopez-Bigas lab: https://twitter.com/nlbigas/status/1295351606855245825/photo/1.